src/app/components/drawer/Projects/ProjectsComponent.js

```diff
@@ -339,17 +339,39 @@ const ProjectsComponent = ({
               </ListItemText>
             </ListItem> :
             <>
-              {feeds.sort((a, b) => (a?.title?.localeCompare(b.title))).map(feed => (
-                <ProjectsListItem
-                  key={feed.id}
-                  routePrefix="feed"
-                  routeSuffix="/feed"
-                  project={feed}
-                  teamSlug={team.slug}
-                  onClick={handleClick}
-                  isActive={isActive('feed', feed.dbid)}
-                />
-              ))}
+              {feeds.sort((a, b) => (a?.title?.localeCompare(b.title))).map((feed) => {
+                let itemProps = {};
+                let itemType = null;
+                switch (feed.type) {
+                // Feeds created by the workspace
+                case 'Feed':
+                  itemProps = { routePrefix: 'feed', routeSuffix: '/feed' };
+                  itemType = 'feed';
+                  break;
+                // Feeds not created by the workspace, but joined upon invitation
+                case 'FeedTeam':
+                  itemProps = { routePrefix: 'feed-team' };
```

Comment:
Looks good in general. I'll just see if it's feasible to change this prefix to be simply `feed`. I'm thinking we could have something like `checkmedia.org/teamslug/feed/id` and keep the `feed-team` entity transparent

Reply:
Sounds good to me if you prefer to find another way to differentiate them in the component, @amoedoamorim , because then the route will be the same for `Feed` and `FeedTeam`. OK for you?

Reply:
Done.